src/components/SelectAsync/index.tsx

```diff
@@ -0,0 +1,131 @@
+/*
+ * Copyright (c) 2022 AccelByte Inc. All Rights Reserved.
+ * This is licensed software from AccelByte Inc, for limitations
+ * and restrictions contact your company contract manager.
+ */
+import React from "react";
+import { SelectProps as AkSelectProps, AsyncSelect, MenuListComponentProps } from "@atlaskit/select";
+import { components, FormatOptionLabelMeta, OptionTypeBase } from "react-select";
+import { SelectOption } from "../../types";
+import "../Select/index.scss";
+
+export const reactSelectComponents = components;
+export type ReactSelectFormatOptionLabelMeta<
```

Comment:
we already FormatOptionLabelMeta in AP. if exporting this is to remove import from react-select in AP, i think we can replace it with the FormatOptionLabelMeta we define in AP